docs/STREAMLINE-INTEGRATION.md

```
@@ -0,0 +1,322 @@
+A reusable `<Illustration>` component located at:
+- `src/components/ui/Illustration/Illustration.tsx`
+
+**Features:**
+- Automatic color theming (black/white in light mode, white/black in dark mode)
+- Responsive sizing with 5 variants: `sm`, `md`, `lg`, `xl`, `hero`
+- Custom color override support
+- Full accessibility support
+- Mobile-responsive with proper scaling
+
+**Usage:**
+```tsx
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)